build.gradle

```diff
@@ -254,7 +254,7 @@ project(':core') {
 
   dependencies {
     compile project(':clients')
-    compile "$slf4jlog4j"
+    compile project(':log4j-appender')
```

Comment:
Log4j-appender needs a separate jar because it is often used in apps directly (as a producer) and we don't want them to use core.

Why does core depend on it is a good question :)